pkg/storaged/stratis-details.jsx

```diff
@@ -185,10 +197,29 @@ export const StratisPoolDetails = ({ client, pool }) => {
                       pool.ClevisInfo[0] && // pool has consistent clevis config
                       (!pool.ClevisInfo[1][0] || pool.ClevisInfo[1][1][0] == "tang")); // not bound or bound to "tang"
     const tang_url = can_tang && pool.ClevisInfo[1][0] ? JSON.parse(pool.ClevisInfo[1][1][1]).url : null;
+    const blockdevs = client.stratis_pool_blockdevs[pool.path] || [];
 
     const forced_options = ["x-systemd.requires=stratis-fstab-setup@" + pool.Uuid + ".service"];
     const managed_fsys_sizes = client.features.stratis_managed_fsys_sizes && !pool.Overprovisioning;
 
+    function grow_blockdevs() {
+        return for_each_async(blockdevs, bd => pool.GrowPhysicalDevice(bd.Uuid));
+    }
+
+    let alert = null;
+    if (client.features.stratis_grow_blockdevs &&
+        blockdevs.some(bd => bd.NewPhysicalSize[0] && Number(bd.NewPhysicalSize[1]) > Number(bd.TotalPhysicalSize))) {
+        alert = (<Alert key="unused-space"
+                        isInline
+                        variant="warning"
+                        title={_("This pool does not use all the space on its block devices.")}>
+            {_("Some block devices of this pool have grown in size after the pool was created. The pool can be safely grown to use the newly available space.")}
+            <div className="storage_alert_action_buttons">
+                <StorageButton onClick={grow_blockdevs}>{_("Grow the pool to take all space")}</StorageButton>
+            </div>
+        </Alert>);
```

Comment:
These 10 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19079-20230825-082034-fda8c7cb-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)